src/views/Housing/utils/staffType/staffType.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Utility function to determine the staff type based on the building code.
+ * @returns {string} - Returns 'RA' for Residence Assistant or 'AC' for Apartment Coordinator based on the building code.
+ */
+export const staffType = {
+  BRO: 'AC',
+  CHA: 'RA',
+  EVN: 'RA',
+  FER: 'RA',
```

Comment:
I have a vague memory that there are apartments in the basement of Ferrin.  Is that true?  Do they have an AC?